test/e2e/runner/wait.go

```diff
@@ -30,5 +30,5 @@ func WaitUntil(testnet *e2e.Testnet, height int64) error {
 // waitingTime estimates how long it should take for a node to reach the height.
 // More nodes in a network implies we may expect a slower network and may have to wait longer.
 func waitingTime(nodes int) time.Duration {
-	return time.Duration(30+(nodes*4)) * time.Second
+	return time.Duration(30+(nodes*120)) * time.Second
```

Comment:
Seems a bit dramatic to multiply by 120. If this work reliably I'd suggest something like:
```suggestion
	return time.Duration(60+(nodes*6)) * time.Second
```
On the other hand, we have time I guess :D

Reply:
I was just trying something insane to get the tests to pass. This worked on an identical branch here https://github.com/evan-forbes/lazyledger-core/pull/37. Those e2e pass consistently, for some reason these still don't.